app/views/steps/application/previous_proceedings/edit.html.erb

```diff
@@ -7,11 +7,21 @@
 
     <h1 class="heading-xlarge gv-u-heading-xxlarge"><%=t '.heading' %></h1>
 
-    <p class="lede gv-u-text-lede"><%=t '.lead_text' %></p>
```

Comment:
Steph, this step needs to stay as was before, as it was already matching in the prototype.
The copy you introduced here correspond to another step:
In the prototype it is: http://localhost:3001/proceedings-protection-check/preview
In the real app it is: http://localhost:3000/steps/miam/child_protection_cases